e2e/e2e_test.go

```diff
@@ -284,3 +566,39 @@ func inRange(cidr string, ip string) error {
 
 	return fmt.Errorf("ip [%s] is NOT in range %s", ip, cidr)
 }
+
+func runIpReconciler(kubeconfigPath string) error {
+	var err error
+	var ipReconcileLoop *reconciler.ReconcileLooper
+
+	reconcilerTimeout := defaultReconcilerTimeout
+	logLevel := "error"
+
+	logging.SetLogLevel(logLevel)
+
+	if kubeconfigPath == "" {
+		ipReconcileLoop, err = reconciler.NewReconcileLooper(context.Background(), reconcilerTimeout)
+	} else {
+		ipReconcileLoop, err = reconciler.NewReconcileLooperWithKubeconfig(context.Background(), kubeconfigPath, reconcilerTimeout)
+	}
```

Comment:
no need for the else, right ?... you know you're not running in a pod, and as such, must pass it the path to the kubeconfig.